plugins/usbdmx/WidgetFactory.h

```diff
@@ -244,7 +269,6 @@ bool BaseWidgetFactory<WidgetType>::AddWidget(WidgetObserver *observer,
 template <typename WidgetType>
 void BaseWidgetFactory<WidgetType>::DeviceRemoved(WidgetObserver *observer,
                                                   libusb_device *usb_device) {
-  OLA_INFO << "Removing device " << usb_device;
```

Comment:
We didn't have this previously, I just added it as I was writing the plugin in accidentally left it in.